ledger/catchpointtracker.go

```diff
@@ -1593,16 +1593,52 @@ func (ct *catchpointTracker) accountsInitializeHashes(ctx context.Context, tx *s
 		// if anything goes wrong, it will still get rolled back.
 		_, err = trie.Evict(true)
 		if err != nil {
-			return fmt.Errorf("accountsInitialize was unable to commit changes to trie: %v", err)
+			return fmt.Errorf("initializeHashes was unable to commit changes to trie: %v", err)
+		}
+
+		// Now add the kvstore hashes
+		// CR Question: Does this neeed ordering? I don't see why it would, but
+		// the account iterator is explicitly ordered for some reason.
+		kvs, err := tx.QueryContext(ctx, "SELECT key, value FROM kvstore")
+		if err != nil {
+			return err
+		}
+		for kvs.Next() {
+			var k []byte
+			var v []byte
+			err := kvs.Scan(&k, &v)
+			if err != nil {
+				return err
+			}
+			hash := kvHashBuilderV6(string(k), v)
+			added, err := trie.Add(hash)
+			if err != nil {
+				return fmt.Errorf("initializeHashes was unable to add kv (key=%s) to trie: %v", k, err)
+			}
+			if !added {
+				if err != nil {
+					ct.log.Warnf("initializeHashes attempted to add duplicate kv hash '%s' to merkle trie for key %s : %v", hex.EncodeToString(hash), k, err)
+				} else {
+					ct.log.Warnf("initializeHashes attempted to add duplicate kv hash '%s' to merkle trie for key %s :  %v", hex.EncodeToString(hash), k, err)
+				}
```

Comment:
the messages are identical, the second has one more space

Reply:
fixing. no point in wrapping the nil error.